src/thunks/compareResultsThunk.ts

```diff
@@ -10,16 +10,20 @@ export const fetchCompareResults = createAsyncThunk<
     baseRev: string;
     newRepo: Repository['name'];
     newRev: string;
+    framework: string;
   },
   { rejectValue: string }
 >(
   'compareResults/fetchCompareResults',
-  async ({ baseRepo, baseRev, newRepo, newRev }, { rejectWithValue }) => {
+  async (
+    { baseRepo, baseRev, newRepo, newRev, framework },
+    { rejectWithValue },
+  ) => {
     let response;
     try {
       //Note: We can now select and add the framework in the url
       response = await fetch(
-        `${treeherderBaseURL}/api/perfcompare/results/?base_repository=${baseRepo}&base_revision=${baseRev}&new_repository=${newRepo}&new_revision=${newRev}&framework=1&interval=86400&no_subtests=true`,
+        `${treeherderBaseURL}/api/perfcompare/results/?base_repository=${baseRepo}&base_revision=${baseRev}&new_repository=${newRepo}&new_revision=${newRev}&framework=${framework}&interval=86400&no_subtests=true`,
```

Comment:
All these values are coming from the URL, so they can be controlled by somebody. Even if the server itself is public, I think it would be good to sanitize this better. We can do it simply with this code:
```js
const searchParams = new URLSearchParams({
  base_repository: baseRepo,
  base_revision: baseRev,
  new_repository: newRepo,
  new_revision: newRev,
  framework,
  interval: 86400,
  no_subtests: "true",
});
const url = `${treeherderBaseURL}/api/perfcompare/results/?${searchParams.toString()}`;
```
And then if any of the variables contain bad characters, they will be properly encoded.